api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -160,12 +160,22 @@ message GetPaymentStateReply {
   // global payment vault parameters
   PaymentGlobalParams payment_global_params = 1;
   // off-chain account reservation usage records
+  // Should be empty if reservation.quorum_numbers is empty (i.e. no reservation exists for the account).
   repeated PeriodRecord period_records = 2;
   // on-chain account reservation setting
   Reservation reservation = 3;
-  // off-chain on-demand payment usage
+  // off-chain on-demand payment usage.
+  // The bytes are parsed to a big.Int value.
+  // This value should always be <= cumulative_payment, as the disperser cumulative_payment kept offchain is only periodically updated onchain.
+  // See [common.v2.PaymentHeader.cumulative_payment] for more details.
+  //
+  // This value should only be nonzero for the EigenLabs disperser, as it is the only disperser that supports on-demand payments currently.
+  // Future work will support decentralized on-demand dispersals.
   bytes cumulative_payment = 4;
   // on-chain on-demand payment deposited
+  // The bytes are parsed to a big.Int value.
+  // This value should always be <= cumulative_payment, as the disperser cumulative_payment kept offchain is only periodically updated onchain.
```

Comment:
we currently don't have payment updates on-chain at all, still something to add in the future. It would look something like disperser sees a request containing payment header, account id, and signature. disperser submits the message onchain, contract verifies, update cumulative_payment_used which would always be `onchain_cumulative_payment_used <=  onchain_cumulative_payment`. then the field above `cumulative_payment` must be within these onchain two ranges